package.json

```diff
@@ -50,6 +50,7 @@
     "dotenv": "^2.0.0",
     "es6-promise": "^4.0.5",
     "express": "^4.14.0",
+    "express-healthcheck": "^0.1.0",
```

Comment:
Shouldn't it show up in npm-shrinkwrap.json as well?

Reply:
Oops, yes.  Done.